massgen/subagent/manager.py

```
@@ -238,9 +243,9 @@ def _validate_runtime_configuration(self) -> None:
                 "Invalid subagent_runtime_fallback_mode: " f"'{self._subagent_runtime_fallback_mode}'. Must be one of [None, 'inherited']",
             )
 
-        if self._subagent_runtime_mode != "isolated" and self._subagent_runtime_fallback_mode is not None:
+        if self._subagent_runtime_mode not in {"isolated", "delegated"} and self._subagent_runtime_fallback_mode is not None:
             raise ValueError(
-                "subagent_runtime_fallback_mode is only valid when subagent_runtime_mode is 'isolated'",
+                "subagent_runtime_fallback_mode is only valid when subagent_runtime_mode is 'isolated' or 'delegated'",
             )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)